DnDTests/Services/UtilsServiceTest.cs

```diff
@@ -84,55 +89,139 @@ public void ValueToAttributeModifier_EqualToTen_ReturnsString()
             Assert.That(actual, Is.EqualTo(expected));
         }
 
-        [Test]
-        public void ModifyCapabilities_ReturnsCapabilityList()
+        [TestCase(CharacterAttributes.STR)]
+        [TestCase(CharacterAttributes.DEX)]
+        [TestCase(CharacterAttributes.CON)]
+        [TestCase(CharacterAttributes.INT)]
+        [TestCase(CharacterAttributes.WIS)]
+        [TestCase(CharacterAttributes.CHA)]
```

Comment:
you can do this iterating over the character attributes witth a testSource so this way you would not need to update this test if you include more characters attributes